src/binary/binary_writer.rs

```diff
@@ -201,10 +201,10 @@ impl<W: Write> IonWriter for BinaryWriter<W> {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::ion_reader::IonReader;
     use crate::reader::ReaderBuilder;
-    use crate::stream_reader::IonReader;
```

Comment:
🗺️ I renamed the file `stream_reader.rs` to `ion_reader.rs`, a long overdue change. The only thing in that file is the trait `IonReader`.